src/setupTests.ts

```diff
@@ -87,7 +87,7 @@ jest.mock("@demo/core/constants/appConfig", () => ({
 		NL_TOGGLE: "nl"
 	},
 	MAP_RESOURCES: {
-		IMPERIAL_COUNTRIES: ["US", "GB", "LR", "MM"],
+		IMPERIAL_COUNTRIES: ["US", "LR", "MM"],
```

Comment:
Same here, GB should still default to imperial.

Reply:
resolved